docs/eigenda/overview.md

```diff
@@ -4,8 +4,8 @@ title: EigenDA Overview
 ---
 
 EigenDA is a data availability store made by
-[EigenLabs](https://www.eigenlayer.xyz/about) and built on top of
-[EigenLayer](../eigenlayer/overview/README.md), currently on mainnet since Q2 2024.
```

Comment:
I was previously getting 404 on the linked Eigenlayer overview page when I created the PR, now appears to work as intended. 

Matter of personal preference then, wether going to the main website or to the docs.

Reply:
The Eigen Labs link is currently inaccurate

Reply:
> I was previously getting 404 on the linked Eigenlayer overview page when I created the PR, now appears to work as intended.
> 
> Matter of personal preference then, wether going to the main website or to the docs.

Linking to the docs makes more sense here since it's for a technical audience

Reply:
Reverted back to the original link in most recent commit